pkg/kyverno/test/test_command.go

```diff
@@ -397,9 +397,9 @@ func testCommandExecute(dirPath []string, fileName string, gitBranch string, tes
 				continue
 			}
 
-			if strings.Contains(file.Name(), fileName) || strings.Contains(file.Name(), "test.yaml") {
+			if strings.Contains(file.Name(), fileName) || strings.HasSuffix(file.Name(), "test.yaml") {
```

Comment:
cc: @chipzoller, @realshuting  can you verify that we want to match test file names exactly and not a substring of the filename for the CLI?

Reply:
@AidanDelaney could you please update the PR accordingly? Also the DCO seems to be failing.

Reply:
@samj1912 @chipzoller This PR is for kyverno 1.7.0. so we can removed logic of test.yaml as we have already plan for deprecate test.yaml in 1.7.0.
we can only look for kyverno-test.yaml in dir's.
any thoughts?

Reply:
I think we also need a bug fix for 1.6.x here since it has incorrect behavior and discovers unintended files as test files and fails the CLI. We can follow up with a PR that removes test.yaml in 1.7.0 so that we can cherry pick this specific PR to 1.6.x. Would that be okay?

Reply:
Yup. we can do that.

Reply:
I've made the check tighter.  I also got the DCO to pass by sync'ing my author and signoff id's.